extensions/sns/extension.json

```diff
@@ -28,6 +28,29 @@
     "deploy": {
       "about": "Subcommand for creating an SNS."
     },
+    "prepare-canisters": {
+      "about": "Subcommand for preparing dapp canister(s) for 1-proposal SNS creation.",
+      "subcommands": {
+        "add-nns-root": {
+          "about": "Subcommand for adding NNS Root as a co-controller of the dapp canister.",
+          "args": {
+            "canister_id": {
+              "about": "CanisterId of a dapp canister your current dfx identity controls.",
+              "long": "canister-id"
+            }
+          }
+        },
+        "remove-nns-root": {
+          "about": "Subcommand for removing NNS Root as a co-controller of the dapp canister.",
+          "args": {
+            "canister_id": {
+              "about": "CanisterId of a dapp canister your current dfx identity controls.",
+              "long": "canister-id"
```

Comment:
```suggestion
              "about": "CanisterId of a dapp canister your current dfx identity controls."
```

Reply:
it also allows you to parse multiple values with 
```json
      "canister_id": {
              "about": "CanisterId of a dapp canister your current dfx identity controls.",
              "multiple": true
```

Reply:
there is a bug, because currently, it will let you parse multiple values weather you specify `multiple: true` or not (of `false), but I'm working on a fix

Reply:
Done! Thanks for this!